src/server/routes/collection.js

```diff
@@ -70,6 +70,12 @@ function getEntityRelations(entityType) {
 		'area'
 	];
 
+	const seriesRelations = [
+		'defaultAlias.language',
+		'disambiguation',
+		'seriesOrderingType'
+	];
```

Comment:
As a possible future improvement, we could think of a way to load the number of items in a series from the ORM model.
That would require loading related relationships through the relationshipSet and filtering them by relationship type, which would probably be quite a costly process.
Let's keep it in mind for another day.